src/bootstrap.scripts.loader.js

```diff
@@ -0,0 +1,16 @@
+/* eslint func-names: 0 */
+
+import processModules from './utils/processModules';
+
+module.exports = function() {
+  if (this.cacheable) this.cacheable();
+
+  const config = global.__BOOTSTRAP_CONFIG__;
```

Comment:
It's because I need to pass normalized config to styles and scripts loader. If I pass it as stringified params, it slows down build: in common case when all bootstrap parts are enabled build time increases from ~2 sec to ~22 sec. As there is no way to pass non-serialized JS object to loader, I made it global.